### DISCUSSION TITLE
Sync with gitlab

### DISCUSSION THREAD

**Discussion by gonuzov** [six7/figma-tokens#1964] [General]
Hi.
The shared library with branded token sets in Figma Tokens from the main DS file is synchronized with Gitlab. With Gitlab we pull token values back into Figma Tokens but into a "different" file; In theory, when we apply a branded token set from the plugin, we should have components repainted in the "other" file, but it doesn't happen. What could be the problem?
The youtube video shows that the token sets are synced with gitlab individually and only one set is pulled into the figma token in the "other" file. In that case, it is not clear how to synchronize token sets individually with gitlab. I will be glad to get feedback

**Comment by six7** [comment]
Hard to guess what could go wrong here without seeing the file, but could be that you have styles applied which won't change unless you change the styles or swap library to the new style library
